src/ui/reducers/timeline.ts

```diff
@@ -71,7 +71,10 @@ const timelineSlice = createSlice({
   },
 });
 
+// If abs(A - B) > EPSILON, then A and B are considered different.
+// If abs(A - B) < EPSILON, then A and B are considered equal.
 const EPSILON = 0.0001;
+const lessThan = (a: number, b: number) => b - a > EPSILON;
```

Comment:
I feel kind of stupid but why is this named "less than" ? isn't it actually checking if the delta is greater than ?

Reply:
It's checking if A is less than B, taking epsilon into account. So it will also return false if B is greater than A but the difference is larger than epsilon.

Reply:
would it be better to name it like `definitelyLessThan`?

Reply:
I think the thing that confused me is that it's checking if the delta between a and b is greater than a minimal threshold.

I don't care. It doesn't matter. I was just confused by the name.

The thing I care about is the question above.